src/endpoints/event.endpoint.js

```diff
@@ -121,7 +121,7 @@ const deleteEvent = (req, res) => {
         for (let i = 0; events.length > i; i++) {
```

Comment:
@arielcamus Hm, yes! I didn't know JS had dictionaries lmao

Nonetheless,now that we've implemented all this logic with Mongo, we just use a findByIdAndRemove, or something like that, so it's pretty straight forward :)